src/theme/components/searchBar.ts

```diff
@@ -10,14 +10,29 @@ const SearchBar = defineMultiStyleConfig({
       base: "xs",
       md: "auto",
     },
-    flexFlow: {
-      base: "column nowrap",
-      md: "row nowrap",
-    },
     ".textInput": {
-      flexGrow: "1",
+      flexGrow: 1,
+      " div > input": {
+        borderRightRadius: 0,
+        borderLeftRadius: "sm",
+      },
```

Comment:
This is overriding the logic that sets the left radius to `0` when the select element is enabled and the left radius is now rounded when the select element is enabled.  Make sure the left radius is `0` when the select element is enabled.